horizon/factdb/client.py

```diff
@@ -39,6 +43,11 @@ async def check_healthy(self) -> bool:
     async def check_ready(self) -> bool:
         return self._backup_loaded or await self.policy_store.is_ready()
 
+    def _init_fast_api_app(self) -> FastAPI:
+        # Called at the of OPALClient.__init__
```

Comment:
fells like there is a word missing in the comment, `Called at the ? of the OPALClient.__init__ `

Reply:
The "of" is unnecessary 😬